Classes/Backend/Form/Element/InputText.php

```diff
@@ -48,7 +48,6 @@ public function render() {
 	public function renderHtml($name, $value, $options) {
 		$width = $options['width'];
 		// TODO: das Feld beachten!
-	 	$maxlength = $options['maxlength'];
```

Comment:
You are right, unused code schould be removed.
The todo can be moved into a Issue or should receive more information about the field to implement soon.